src/system/data-transfer-tax-rates.md

```diff
@@ -2,7 +2,10 @@
 title: Import/Export Tax Rates
 ---
 
-If you conduct business in several states and ship a large quantity of product, entering tax rates manually can be very time consuming. It is faster and more efficient to download tax rates by ZIP code and import them into Magento. The following example shows how to import a set of California tax rates downloaded from the [Avalara][1]{:target="_blank"} website. Avalara is a [Magento Technology Partner][2]{:target="_blank"} and provides [tax rate tables][3]{:target="_blank"} for every ZIP code in the United States, which you can download at no charge.
+If you conduct business in several states and ship a large quantity of product, entering tax rates manually can be very time consuming. It is faster and more efficient to download tax rates by ZIP code and import them into Magento. The following example shows how to import a set of state-specific tax rates downloaded from the a trusted source. Avalara provides [tax rate tables](http://www.avalara.com/taxrates/en/download-tax-tables.html){:target="_blank"}, that you can download at no charge, for every ZIP code in the United States.
+
+{:.bs-callout .bs-callout-info}
+If you are interested in automating your sales and use tax compliance and reporting, Magento versions 2.2.4+ and 2.3.0+ offer [Vertex Cloud]({% link tax/vertex.md %}){:target=“_blank”} as a trusted, built-in option. You can also find Magento-trusted options on the [Magento Partners](https://partners.magento.com/portal/directory/?_token=_vJAasZxq5QA0moyZVfX1SczSr3WWWRp&query=sales+tax){:target="_blank"} site.
```

Comment:
Is it necessary to mention 2.2.x in 2.3 documentation?

Reply:
it was a request from the vbe manager. i didn't have it at first.